test/Frontend/enable_builtin_module.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: not %target-swift-frontend -typecheck -show-diagnostics-after-fatal -D BUILTIN_IMPORT %s 2>&1 | %FileCheck -check-prefix CHECK-NO-BUILTIN %s
+// RUN: not %target-swift-frontend -typecheck -show-diagnostics-after-fatal  -enable-builtin-module %s 2>&1 | %FileCheck -check-prefix CHECK-NO-BUILTIN-IMPORT %s
+// RUN: %target-swift-frontend -typecheck -enable-builtin-module -D BUILTIN_IMPORT %s
+
+// CHECK-NO-BUILTIN: no such module 'Builtin'
+
+#if BUILTIN_IMPORT
+import Builtin
+#endif
+
+// CHECK-NO-BUILTIN: cannot find type 'Builtin' in scope
+// CHECK-NO-BUILTIN-IMPORT: cannot find type 'Builtin' in scope
+
+func something(_: Builtin.RawPointer) {
```

Comment:
Does explicitly importing `Builtin` still require qualifying `RawPointer` with the module name?

(Note: that would not be a problem; I would actually prefer if that was the case.) 
((And in fact it ought to be an option for regular imports, too.))

Reply:
Yes, using `Builtin` still requires that all APIs are qualified.